### PR TITLE
Make blog post more customer facing

### PR DESCRIPTION
**Pull request by samay-sharma**

### CODE REVIEW

File: blog/2023-07-05-tembo-manifesto/index.md [hunk 1]
```diff
@@ -94,36 +86,19 @@ Stacks are pre-built, use case-specific configurations of Postgres, enabling you
...
-- Need to recruit top Postgres committers to the company in order to obtain legitimacy within the Postgres community.
-- Databases are the most complex layer of the stack; takes more time than traditional developer/data services.
+The channel of distribution is a fully-managed cloud service, providing enterprise-level hybrid multi-cloud solutions.
 
 ## Bottom Line
 
-Postgres is the most universally loved databases for developers, teams, and enterprises of all sizes. Tembo can become the new de facto standard for deploying and managing databases for developers and companies of all sizes, changing the paradigm of how developers interact and build databases, and unlocking the power of Postgres to all developers.
\ No newline at end of file
+Postgres is the most universally loved databases for developers, teams, and enterprises of all sizes. Tembo can become the new de facto standard for deploying and managing databases for developers and companies of all sizes, changing the paradigm of how developers interact and build databases, and unlocking the power of Postgres to all developers.
```
Comment (aijamalnk) [inline blog/2023-07-05-tembo-manifesto/index.md:104]:
```suggestion
Postgres is the most universally loved database for developers, teams, and enterprises of all sizes. Tembo can become the new de facto standard for deploying and managing databases for developers and companies of all sizes, changing the paradigm of how developers interact and build databases, and unlocking the power of Postgres to all developers.
```

File: blog/2023-07-05-tembo-manifesto/index.md [hunk 2]
```diff
@@ -94,36 +86,19 @@ Stacks are pre-built, use case-specific configurations of Postgres, enabling you
...
+## Users
 
-Initial ICP targets application developers using or intending to use self-managed Postgres deployments, specifically those interested in maintaining the native open-source version. Tembo replaces self-managed Postgres clusters in the cloud or on-prem.
+Initial users for our platform are application developers using or intending to use self-managed Postgres deployments, specifically those interested in maintaining the native open-source version. Tembo replaces self-managed Postgres clusters in the cloud or on-prem.
 
-Our market entry focuses on two key propositions:
+Our inital set of users are interested in one of two key propositions:
 
-1. Developer-first workflows, facilitating complete data model lifecycle control.
+1. Developer-first workflows for use-cases, facilitating complete data model lifecycle control.
```
Comment (aijamalnk) [inline blog/2023-07-05-tembo-manifesto/index.md:95]:
```suggestion
1. Developer-first workflows, tailored to specific use-cases, facilitating complete data model lifecycle control.
```

File: blog/2023-07-05-tembo-manifesto/index.md [hunk 3]
```diff
@@ -94,36 +86,19 @@ Stacks are pre-built, use case-specific configurations of Postgres, enabling you
...
-## Advantages
-
-- Large market continues to grow rapidly with a significant gap in a developer-first database.
-- Impressive team with world-class open source & Postgres cred.
-- Replaces self-managed & home grown solutions with a 10x better experience.
-- Expect the offering to have a very fast TTV and high level of developer love.
-- A large and passionate open-source community could lead to strong growth.
-
-## Risks
+Fast database deployment and automatic migration enable high developer satisfaction and swift value realization. We plan to be aligned with the Postgres open-source principles of the Postgres community.
```
Comment (aijamalnk) [inline blog/2023-07-05-tembo-manifesto/index.md:98]:
```suggestion
Fast database deployment and automatic migration enable high developer satisfaction from having a smooth and quick development experience. This leads to swift value realization.

We plan to be aligned with the Postgres open-source principles of the Postgres community.
```

This one is a bit more adventurous. Feel free to reject.

File: blog/2023-07-05-tembo-manifesto/index.md [hunk 4]
```diff
@@ -94,36 +86,19 @@ Stacks are pre-built, use case-specific configurations of Postgres, enabling you
...
 
-- Lots of Postgres alternatives/competition. Need to break through the noise of the competition.
-- Banking on (1) a radically simpler Postgres DX and (2) extending out the extensions framework and capabilities will become a catalyst for initial usage and growth.
-- Need to recruit top Postgres committers to the company in order to obtain legitimacy within the Postgres community.
-- Databases are the most complex layer of the stack; takes more time than traditional developer/data services.
+The channel of distribution is a fully-managed cloud service, providing enterprise-level hybrid multi-cloud solutions.
```
Comment (aijamalnk) [inline blog/2023-07-05-tembo-manifesto/index.md:100]:
```suggestion
The channel of distribution is a fully-managed cloud service, providing enterprise-level hybrid multi-cloud solutions based on Postgres and its open source ecosystem.
```

File: blog/2023-07-05-tembo-manifesto/index.md [hunk 5]
```diff
@@ -73,11 +65,11 @@ We are building a dev-first, fully-extensible, fully-managed, secure, and scalab
 | --- | --- | --- |
 | Tembo OLTP | Feature-Rich, Optimized Postgres | Other managed Postgres providers |
 | Tembo Vector | Optimized Postgres + pgVector/pg_embedding | Pinecone, ChromaDB |
-| Tembo OLAP | Optimized Postgres + columnar | Snowflake |
+| Tembo Data Warehouse | Optimized Postgres + columnar | Snowflake |
 | Tembo Documents | Optimized Postgres + FerretDB | MongoDB, AWS DocumentDB |
 | Tembo ML | Optimized Postgres + PostgresML | MindsDB |
-| Tembo Messaging | Optimized Postgres + pgmq | Redis |
-| Tembo Search | Optimized Postgres + ZomboDB + ElasticSearch | Elastic |
+| Tembo Messaging | Optimized Postgres + pgmq | Redis, SQS |
```
Comment (aijamalnk) [inline blog/2023-07-05-tembo-manifesto/index.md:71]:
```suggestion
| Tembo Messaging | Optimized Postgres + pgmq | Redis, SQS, RabbitMQ |
```

**Review by aijamalnk — COMMENTED** [review]
Thanks for the edits. I made minor comments.

**Review by ryw — APPROVED** [review]
thanks for edits :)

**Pull request merged**